test/waittask.spec.js

```diff
@@ -247,15 +254,6 @@ module.exports.addTests = function({testRunner, expect, product}) {
       expect(eHandle.executionContext().frame()).toBe(frame2);
     });
 
-    it('should throw if evaluation failed', async({page, server}) => {
```

Comment:
Does this imply that we can never throw now?

Reply:
yep, only for timeouts